src/App.js

```diff
@@ -244,23 +265,27 @@ const App = () => {
                   </Grid>
                   <Grid item xs={12}>
                     <Typography variant="h4">Listings</Typography>
-                    {filteredListings.map((listing) => (
-                      <div
-                        key={listing.id}
-                        onClick={() => handleListingClick(listing)}
-                        style={{ cursor: 'pointer' }}
-                      >
-                        <Paper style={{ display: 'flex', alignItems: 'center', padding: '10px', marginBottom: '10px' }}>
-                          <img src={listing.imageUrl} alt={listing.title} style={{ marginRight: '10px' }} />
-                          <div>
-                            <Typography variant="h6" style={{ fontWeight: 'bold' }}>{listing.title}</Typography>
-                            <Typography variant="subtitle1" style={{ marginBottom: '5px', overflow: 'hidden', textOverflow: 'ellipsis', whiteSpace: 'nowrap', maxWidth: '300px' }}>{listing.description}</Typography>
-                            <Typography variant="subtitle2">Condition: {listing.condition}</Typography>
-                            <Typography variant="subtitle2">Price: ${listing.price}</Typography>
-                          </div>
-                        </Paper>
-                      </div>
-                    ))}
+                    {filteredListings.length > 0 ? (
+                      filteredListings.map((listing) => (
+                        <div
+                          key={listing.id}
+                          onClick={() => handleListingClick(listing)}
+                          style={{ cursor: 'pointer' }}
+                        >
+                          <Paper style={{ display: 'flex', alignItems: 'center', padding: '10px', marginBottom: '10px' }}>
+                            <img src={"https://via.placeholder.com/150"} alt={listing.title} style={{ marginRight: '10px' }} />
```

Comment:
We probably want to replace this placeholder URL with the images you're bringing in now that Griffin added those to the database

Reply:
Yes, I resolved the conflict and kept your previous code.